pages/blog/event-driven-api-documentation-made-simple-clientside-rendering.md

```diff
@@ -0,0 +1,264 @@
+---
+title: Event driven API documentation made simple (Client-Side Rendering).
+date: 2022-08-22T9:43:15+01:00
```

Comment:
@m1ner79 let us release it next Tuesday as there is another blog that we already want to release this Thursday

```suggestion
date: 2022-09-27T06:00:00+01:00
```